shell/browser/api/atom_api_app.cc

```diff
@@ -857,7 +857,6 @@ void App::SetAppLogsPath(util::ErrorThrower thrower,
   } else {
     base::FilePath path;
     if (base::PathService::Get(DIR_USER_DATA, &path)) {
-      path = path.Append(base::FilePath::FromUTF8Unsafe(GetApplicationName()));
```

Comment:
this was adding an extra name directory to the logs:
`~/.config/Electron Tests Main/Electron Tests Main/logs` instead of `~/.config/Electron Tests Main/logs`, which was incorrect and causing the previous test failure as that's not what documentation outlined should be the case.

Reply:
But this is a `semver/major` change because the pre-existing behavior is in 4-2-x, 5-0-x, and 6-0-x

Reply:
agreed that this is a point of discussion - arguably it was previously incorrect which makes this, to some extent, a fix 🤔